PERF_TESTS_WIP.md

```diff
@@ -0,0 +1,39 @@
+`go test -bench=. -run=^B`
+
+LOCAL CLIENT
+
+goos: linux
+goarch: amd64
+pkg: github.com/singlestore-labs/singlestoredb-go-arrow-driver
+cpu: 12th Gen Intel(R) Core(TM) i7-1260P
+
+SINGLEBOX
+BenchmarkReadRegular-16                        1        7997833847 ns/op
+BenchmarkReadArrow-16                          1        8213899218 ns/op
+BenchmarkReadArrowServer-16                    3         440983040 ns/op
+
+
+LOCAL cluster-in-a-box
+BenchmarkReadRegular-16                        1        6325171003 ns/op
+BenchmarkReadParallel-16                       1        9090986287 ns/op
+BenchmarkReadArrow-16                          1        14120213354 ns/op
+BenchmarkReadArrowParallel-16                  1        9390935150 ns/op
+BenchmarkReadArrowServer-16                    4         275127879 ns/op
+
+S-1 S2MS SERVER
+BenchmarkReadRegular-16                        1        4126553651 ns/op
+BenchmarkReadParallel-16                       1        5179306474 ns/op
+BenchmarkReadArrow-16                          1        4404532083 ns/op
+BenchmarkReadArrowParallel-16                  1        4942044003 ns/op
+BenchmarkReadArrowServer-16                    1        3442768146 ns/op
```

Comment:
Why it gives so small improvement in S2MS compared to singlebox and ciab?

Reply:
Most likely because network transmission takes significant amount of time, and the amount of data transferred is pretty much the same.

What's very strange with the current benchmarks: parallel read is slower than regular read. It's not supported with Arrow yet, but we'll need to look into this closer later.